prog-o-meter.py

```diff
@@ -80,19 +80,41 @@ def canvas_layout(self):
     def button_layout(self):
         """Display a button with the text "1 more day!" on the canvas.
         
-        Creates and display a button with the text "1 more day!" with the function add_day() as callback function. If user have already reached their goal, the button is displayed, but is disabled. 
-       
+        Creates and display a button with the text "1 more day!" with the function add_day() as callback function. If user have already reached their goal, the button is displayed, but is disabled.
+        
         Attributes:
             add_day_button: A button with the text "1 more day!", which calls the function add_day
         """
         self.add_day_button = Tk.Button(self.root, text = "1 more day!", command = self.add_day)
         self.add_day_button.pack()
-        if self.days >= self.GOAL:        # Disable add_day_button if goal have been reached
+        if self.days >= self.GOAL:  # Disable add_day_button if goal have been reached
             self.add_day_button.config(state = "disabled")
+        """Display a button with the text "Whoops!" on the canvas.

```

Comment:
The type of documentation you have created here (in line 92-98) is called docstring documentation. Docstrings only belongs right in the beginning of functions, methods and classes. Since you are not creating any of those here, a docstring is not needed. Instead, the information you are adding, should be added to the docstring belonging to the function you are updating, so in this case, the docstring in line 81-87 (which is the documentation of the function button_layout).